scripts/monitor.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# function to send email
+send_alert_email() {
+    to=$1
+    echo "Sending alert message to $to"
+    curl --ssl-reqd \
+  --url 'smtps://smtp.gmail.com:465' \
+  --user 'lindsaywildlifetech@gmail.com:qinaytsnliiumtvq' \
```

Comment:
o no rory :( plz remove the credentials from the PR and create environment variables for these